src/site/content/en/metrics/inp/index.md

```diff
@@ -0,0 +1,286 @@
+
+In the following video, the example on the right gives immediate visual feedback that an image is being fetched from the network. This visual feedback underscores the importance of communicating a result of an interaction.
+
+<figure>
+  {% Video src="video/jL3OLOhcWUQDnR4XjewLBx4e3PC3/vPC97OKo8wX4RtYZDtMQ.mp4", autoplay=true, controls=true, loop=true %}
+  <figcaption>
+    A visual representation of both poor and good responsiveness where a full-sized image is loaded from a thumbnail gallery. On the left, responsiveness is poor because the image has to be downloaded before it is displayed, which involves a considerable delay. On the right, responsiveness is good because a loading indicator appears immediately after requesting the image, and the requested image eventually replaces it.
+  </figcaption>
+</figure>
+
+This article explains how INP works, how to measure it, and offers suggestions for improving it, as good responsiveness is imperative to a good user experience.
+
+## What is INP?
+
+INP is a metric that aims to represent a page's overall _interaction latency_ by selecting one of the single longest interactions that occur when a user visits a page. For pages with less than 50 interactions in total, INP is the interaction with the worst latency. For pages with many interactions, INP is most often the 98th percentile of interaction latency.
+
+{% Aside 'important' %}
+For more information on why the longest interaction isn't selected for every page, read the ["Why not the worst interaction latency?"](/inp/#why-not-the-worst-interaction-latency) section for more details.
+{% endAside %}
+
+An _interaction_ is a set of related input events that fire during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`, all of which may contribute to the overall interaction's _latency_.
+
+A single interaction's latency consists of the single longest duration of any event that is part of the interaction, where the _duration_ is measured from the point at which the user interacted with the page until the next frame is presented after all associated event handlers have executed. The duration is the sum of the following timespans:
+
+- The _input delay_, which is the time between when the user interacts with the page, and when event handlers execute.
+- The _processing time_, which is the total amount of time it takes to execute code in associated event handlers.
+- The _presentation delay_, which is the time between when event handlers have finished executing, and when the browser presents the next frame.
+
+{% Aside 'important' %}
+For more information on exactly how INP is measured, read the ["What's in an interaction?"](/inp/#what's-in-an-interaction) section for more details.
```

Comment:
```suggestion
For more information on exactly how INP is measured, read the [What's in an interaction?](/inp/#what's-in-an-interaction) section for more details.
```

Reply:
linked references don't need quoting, according to https://developers.google.com/style/cross-references?hl=en